dbus/dbus.go

```diff
@@ -148,6 +148,15 @@ func newConnection(createBus func() (*dbus.Conn, error)) (*Conn, error) {
 	return c, nil
 }
 
+// GetProperty returns values of a org.freedesktop.systemd1.Manager interface property
+func (c *Conn) GetProperty(prop string) (string, error) {
```

Comment:
Since the `Conn` is pretty general, I think it might be a little clearer to call this `GetManagerProperty` (c.f. the existing `GetUnitProperty` for example) - does that seem reasonable? Other than that, LGTM